src/view/renderer.js

```diff
@@ -407,6 +406,12 @@ export default class Renderer {
 	 */
 	_updateText( viewText, options ) {
 		const domText = this.domConverter.findCorrespondingDomText( viewText );
+
+		// If this is a new text node and it is not in DOM, it will be created and handled in `_updateChildren`.
+		if ( !domText ) {
```

Comment:
Aaand it turned out we can't make this change ;> We'll also need to make a step back and remove the fix for subsequent text nodes updating. It seems that we may not be able to finalise it now because of how the text nodes are mapped. 

We can neither update all text nodes before children are updated because changes in children might've changed the indexes of text nodes (so updating text nodes would fail). Changing the order to 1) update children, 2) update text nodes doesn't help too because in the following case, the whole paragraph is updated first and then text nodes are not:

1. `<p>x<strong> b</strong></p>`
2. Type space after "x".
3. Paragraph's children get updated (because for some reason the browser mutated text nodes too much when typing the space). 
4. The `"x "` text node is up to date when it comes to updating text nodes, so we don't mark `" b"` to be updated.